tests/cache/cache.c

```diff
@@ -0,0 +1,378 @@
+/*
+ * rv32emu is freely redistributable under the MIT License. See the file
+ * "LICENSE" for information on usage and redistribution of this file.
+ */
+
+#include "cache.h"
```

Comment:
Always ensure system headers prior to local ones. That is, move `cache.h` at the end of inclusion.